spec/services/import_transactions_spec.rb

```diff
@@ -398,20 +379,20 @@
     end
 
     it "assigns each transaction to the correct report" do
-      expect(report.transactions.map(&:description).sort).to eq([
-        "Christmas Donation",
-        "Fees for Q3",
-        "Rent Payments",
+      expect(report.transactions.map(&:value).sort).to eq([
+        50,
+        150,
+        5000,
```

Comment:
I'm wondering if there's some way we could make this test clearer -- previously the `description` functioned as a way of identifying the different transaction rows from the CSV, and it's harder to parse using just the `value` numbers.

Reply:
I've wondered that too. Is it worth giving the factoried activities custom names, so we can predictably test descriptions? (Otherwise it will be random Lorem ipsum.)

Reply:
Yes, that sounds good -- so we could put `meaningful_name.value` in the assertions instead of `50`.

Reply:
Apologies it's taken me this long to get to this @jcoglan . Is the new approach better?